.github/workflows/build_macos.yml

```diff
@@ -11,11 +11,11 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        configuration: [Debug, Release]
+        configuration: [debug, release]
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: 'true'
-      - run: cmake -S . -B build -G "Xcode"
-      - run: cmake --build build --config ${{matrix.configuration}} 
-      
\ No newline at end of file
+
+      - name: Build
+        run: ./build_linux.sh ${{matrix.configuration}}
```

Comment:
It should be `run: ./build_macOS.sh`

Reply:
my bad.

Reply:
I've accidentally closed this pr, and could not reopen it. I'll submit a new one then.